src/System.Management.Automation/engine/CommonCommandParameters.cs

```diff
@@ -232,6 +232,22 @@ public string PipelineVariable
             set { _commandRuntime.PipelineVariable = value; }
```

Comment:
@KirkMunro, your last commit had 7 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Verify approved aliases list.All approved aliases present (no new aliases added, no aliases removed)
```powershell
Expected $null or empty, but got @(@{InputObject=gcb -> Get-Clipboard; SideIndicator==>}, @{InputObject=scb -> Set-Clipboard; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 553
553:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @('Get-Clipboard', 'Set-Clipboard').
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 584
584:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Update-Help; ConfirmImpact=Medium; SideIndicator==>}, @{Name=Write-Warning; ConfirmImpact=None; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can enable breakpoints
```powershell
Exception calling "EnableBreakpoint" with "1" argument(s): "The background process reported an error with the following message: Remote transport error: ."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 106
106:                 $bp = $jobRunspace.Debugger.EnableBreakpoint($bp)                

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can remove breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 120
120:             foreach ($bp in $jobRunspace.Debugger.GetBreakpoints()) {

```